napari/components/add_layers_mixin.py

```diff
@@ -150,6 +152,17 @@ def add_image(
             Translation values for the layer. If a list then must be a list of
             tuples of float with the same length as the axis that is being
             expanded as channels.
+        rotate : float, 3-tuple of float, n-D array or list.
+            If a float convert into a 2D rotate matrix using that value as an
+            angle. If 3-tuple convert into a 3D rotate matrix, rolling a yaw,
+            pitch, roll convention. Otherwise assume an nD rotate. Angle
+            conversion are done either using degrees or radians depending on the
+            degrees boolean parameter. If a list then must have same length as
```

Comment:
Where is the degrees boolean parameter? How do I use it?

Reply:
Edited to add: ah, I see `degrees` is a kwarg for `Affine`, but this isn't exposed to any of our higher level `add_*` methods instead.

Reply:
Yes, that's right! Good catch, I just copied and pasted doc strings. We could expose here too, but i worry about API explosion a bit. Curious what people recommend.

Reply:
I'm happy for you to fix it and always use one or the other (degrees/radians). That might simplify things.

Reply:
I like that idea, I can drop the reference to the `degrees` parameter in the `add_*` methods to keep things simple. I might leave it in the `Affine` class just in case we want to expose it in the future

Reply:
Maybe just write in the docstring what coordinate system you choose. I think `degrees=True` is the default?

Reply:
done